vms/rpcchainvm/vm_server.go

```diff
@@ -216,6 +216,8 @@ func (vm *VMServer) Initialize(ctx context.Context, req *vmpb.InitializeRequest)
 				}
 				// Nothing to do with the error within the goroutine
 				_ = msgClient.Notify(msg)
+			case <-ctx.Done():
+				return
```

Comment:
I don't think we would want to exit this loop when the `Initialize` context is cancelled - This `goroutine` should run until the VM is shutdown.